entries/f/fubo.tv.json

```
@@ -0,0 +1,13 @@
+{
+    "fuboTV": {
+        "domain": "fubo.tv",
+        "contact": {
+            "twitter": "fuboTV",
+            "facebook": "fuboTV",
+            "email": "support@fubo.tv"
+        },
+        "keywords": [
+            "entertainment"
+        ]
```

According to [this page](https://support.fubo.tv/hc/en-us/articles/115002979407-Is-fuboTV-available-in-my-country) only these countries are served:
```suggestion
        ],
        "regions": [
          "ca",
          "es",
          "us"
```